tests/test_all_fields.py

```diff
@@ -47,7 +47,8 @@
     'subscriptions': {
         'test_clock',
         'description',
-        'application'
+        'application',
+        'currency'
```

Comment:
`# we have observed that the SDK object creation returns some new fields intermittently, which are not present in the schema`

If above is the case, should have `must have field lists` instead of `missing field lists` because in future we may see failure for some new intermittent field?

Reply:
Changed the assertion to issubset() so that we dont have to add the fields always